app.py

```diff
@@ -17,28 +14,119 @@
 
 logger = logging.getLogger(__name__)
 
+COLORS_text = ['purple', 'white', 'blue', 'yellow', 'orange', 'black', 'red', 'green', 'rainbow']
+COLORS_icon = ['🟪', '  ', '🟦', '🟨', '🟧', '⬛', '🟥', '🟩',  '🏳️‍🌈']
+COLORS_text_to_icon = {text : icon for text, icon in zip(COLORS_text, COLORS_icon)}
+
+class Deck:
```

Comment:
I think it's better to move `Deck` and `RoutesDeck` to a separate module, it would be easier to read